recipes/mysql/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mysql" %}
+{% set major_minor_version = "5.7" %}
+{% set micro_version = "20" %}
+{% set sha256 = "260582844ac18222ce2826f48b6c7ca387990b19ddb88331af26738b394e42a4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ major_minor_version }}.{{ micro_version }}
+
+source:
+  fn: mysql-boost-{{ major_minor_version }}.{{ micro_version }}.tar.gz
+  url: https://cdn.mysql.com/Downloads/MySQL-{{ major_minor_version }}/mysql-boost-{{ major_minor_version }}.{{ micro_version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: build.sh
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+  run:
```

Comment:
Drop this line if there's nothing there.

Reply:
`openssl` should be [pinned](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L81) at `openssl 1.0.*` in both build and run.

`libedit` should probably be pinned as `libedit 3.1` for now; investigating further.